bin/proving-service/build.rs

```diff
@@ -35,12 +35,21 @@ fn main() -> miette::Result<()> {
 
 /// Copies the proto file from the root proto directory to the proto directory of this crate.
 fn copy_proto_files() -> miette::Result<()> {
-    let src_file = format!("{REPO_PROTO_DIR}/proving_service.proto");
-    let dest_file = format!("{CRATE_PROTO_DIR}/proving_service.proto");
+    let src_files = [
+        format!("{REPO_PROTO_DIR}/proving_service.proto"),
+        format!("{REPO_PROTO_DIR}/worker_status.proto"),
+    ];
+    let dest_files = [
+        format!("{CRATE_PROTO_DIR}/proving_service.proto"),
+        format!("{CRATE_PROTO_DIR}/worker_status.proto"),
+    ];
```

Comment:
nit: This could be reduced to 
```rust
    let files = ["proving_service.proto", "worker_status.proto"];

    // remove and create dirs

    for file in &files {
        let src = format!("{REPO_PROTO_DIR}/{}", file);
        let dest = format!("{CRATE_PROTO_DIR}/{}", file);
        fs::copy(src, dest).into_diagnostic()?;
    }
```